button-merchant/src/main/java/com/usebutton/merchant/PostOrderTask.java

```diff
@@ -37,22 +41,75 @@
     private final String sourceToken;
     private final Order order;
     private final DeviceManager deviceManager;
+    private final ThreadManager threadManager;
+
+    private Getter<Double> retryDelayInterval = new Getter<Double>() {
```

Comment:
Is using the `Getter` class necessary? Can this just be a private method?

Reply:
++